service/src/utils/logger.ts

```diff
@@ -19,6 +21,11 @@ export class Logger {
     }
 
     public log(...args: any[]) {
+        if(!configContext().settings.disableLog) {
```

Comment:
Probably better to put it in the constructor. This will be evaluated every time the log is called.

Reply:
Got it. Thanks for the suggestion.